# metaG-workshop
Main repo for working on the Metagenome workshop

## Tentative Schedule

| Day 1| Day 2| Day 3  |
|:---------|:------|:------|
| Intro to CHTC | Metagenomic Assembly | [Comparative Genomics](https://uw-madison-microbiome-hub.github.io/metaG-comp-genomics/) [ [repo](https://github.com/uw-madison-microbiome-hub/metaG-comp-genomics) ] |
| [Filtering/Wrangling metagenomic data](https://uw-madison-microbiome-hub.github.io/metaG-wrangling/) [ [repo](https://github.com/uw-madison-microbiome-hub/metaG-wrangling) ]| [Binning](https://uw-madison-microbiome-hub.github.io/metaG-bin-analysis/) [ [repo](https://github.com/uw-madison-microbiome-hub/metaG-bin-analysis) ] | Comparative Genomics cont. |
| LUNCH | LUNCH | LUNCH |
| [Mapping to reference databases](https://uw-madison-microbiome-hub.github.io/metaG-ref-analysis/) [ [repo](https://github.com/uw-madison-microbiome-hub/metaG-ref-analysis) ] | QC Bins and Refine | Other analyses with Bins |
| Phylogenetic and functional analysis | Binning Best Practices | Freetime and Discussion | 

## Learning Objectives 

- Explain what analysis you perform with metagenomes/genomes
- Know how to get genomes from metagenomes
- When you map to reference databases vs binning
- Relavent software for each analysis step
- Be able to run jobs on HTC/CHTC
